Migrate to recipe.yaml (CEP 13/14)

zachcp opened the pull request:
Draft PR for recipe.yaml migration per CEP 13/14.

zachcp committed:
Migrate to recipe.yaml (CEP 13/14)

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

zachcp committed:
Fix: Skip validation for rattler-build (recipe.yaml not yet supported)

zachcp committed:
Revert "Fix: Skip validation for rattler-build (recipe.yaml not yet supported)"

This reverts commit 6e43a510c31bc7015e60a67610619026b5b0a741.

zachcp committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.11.30.18.14.46

Other tools:
- conda-build 25.11.0
- rattler-build 0.53.0
- rattler-build-conda-compat 1.4.10

zachcp committed:
Skip validation for recipe.yaml - conda-build validator not yet compatible

zachcp committed:
Add rattler-build and pixi configuration
